frontend/integration-tests/tests/performance.scenario.ts

```diff
@@ -60,13 +60,13 @@ describe('Performance test', () => {
   });
 
   it('downloads new bundle for YAML editor route', async() => {
-    await browser.get(`${appHost}/k8s/ns/openshift-console/deployments`);
+    await browser.get(`${appHost}/k8s/ns/openshift-console/configmaps`);
     await crudView.isLoaded();
 
     const initialChunks = await browser.executeScript<{name: string, size: number}[]>(() => performance.getEntriesByType('resource')
       .filter(({name}) => name.endsWith('.js')));
 
-    await crudView.selectOptionFromGear('console', 'Edit Deployment');
+    await browser.get(`${appHost}/k8s/ns/openshift-console/configmaps/console-config/yaml`);
```

Comment:
I think triggering a new page load with break the initial chunks vs post chunks test below.

cc @alecmerdler

Reply:
```suggestion
    await crudView.selectOptionFromGear('console-config', 'Edit Config Map');
```

Reply:
@spadgett yes, I did try this, but at least on my local system, the config map took a bit to load. The selection failed stating 0 objects in list, and stopped the tests. I could try what you added and assume my system is slower than normal clusters and hope for the best?